packages/fast-components-react-base/src/utilities/resize-observer.ts

```diff
@@ -1,6 +1,6 @@
 import { ResizeObserverEntry } from "./resize-observer-entry";
 
-export type ConstructableResizeObserver = new (
+export type ConstructibleResizeObserver = new (
     callback: ResizeObserverCallback
```

Comment:
what does `new` do in during type definition?

Reply:
I pretty much copied the existing "resizeObserver" implementation assuming it was good...